docs/ossm/ossm2-migration/README.md

```diff
@@ -0,0 +1,20 @@
+# OSSM 2.x migration to 3.x
+
+This document details how to migrate from OSSM 2.x to OSSM 3.x.
+
+## Pre-migration Checklist
+
+Before you begin to migrate your controlplane from OSSM 2.x to 3.x, ensure you have done the following:
+
+- Updated your `SMCP` version to the latest OSSM release.
```

Comment:
I know it's verbose, but we should use `ServiceMeshControlPlane` rather than `SMCP` to be clear. "OSSM" also isn't something we can use in the docs, though it's easy for docs to do a search and replace on it.